src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy

```diff
@@ -232,15 +231,6 @@ class ScalaMultiVersionPlugin implements Plugin<Project> {
 
     private void resolvePomDependencies() {
         project.afterEvaluate {
-            // for projects using the maven plugin
```

Comment:
I am dropping support for the long-deprecated `maven` plugin. Use `maven-publish` instead.

Reply:
Does this change require a major rev?

Reply:
Yep. I bumped it probably as you started reviewing.